prow/config/jobs/istio.yaml

```diff
@@ -431,7 +431,7 @@ jobs:
     - --token-path=/etc/github-token/oauth
     - --cmd=go get github.com/envoyproxy/go-control-plane@main && go mod tidy && make clean gen
     requirements: [github]
-    repos: [istio/istio@master]
+    repos: [istio/test-infra@master]
```

Comment:
Changing to the test-infra repo is correct since it's needed to run the automator script.